lib/recurly.js

```diff
@@ -35,6 +36,10 @@ const defaults = {
   publicKey: '',
   parent: true,
   cors: false,
+  fraud: {
+    dataCollector: false,
```

Comment:
I can see the value in calling it dataColector within the Kount context, then. I'm a bit concerned with moving that into our broad 'fraud' context though. How about this as a compromise?

```js
{
  fraud: {
    kount: { dataCollector: false },
    litle: { sessionId: null }
  }
}
```

That way we're a bit more clear within the space of 'Recurly fraud protection'